cmd/cmd_controller.js

```diff
@@ -419,20 +419,16 @@ class EmbarkController {
 
   ejectWebpack() {
     var fs = require('../lib/core/fs.js');
+    var embarkConfig = fs.embarkPath('lib/modules/pipeline/webpack.config.js');
     var dappConfig = fs.dappPath('webpack.config.js');
-    var embarkConfig = fs.embarkPath('lib/pipeline', 'webpack.config.js');
-    let ext = 1;
-    let dappConfigOld = dappConfig;
-    while (fs.existsSync(dappConfigOld)) {
-      dappConfigOld = dappConfig + `.${ext}`;
-      ext++;
-    }
-    if (dappConfigOld !== dappConfig) {
-      fs.copySync(dappConfig, dappConfigOld);
-    }
-    fs.copySync(embarkConfig, dappConfig);
-    console.log(__('webpack config ejected to: ').dim.yellow);
+    fs.copyPreserve(embarkConfig, dappConfig);
+    console.log(__('webpack config ejected to:').dim.yellow);
     console.log(`${dappConfig}`.green);
+    var embarkOverrides = fs.embarkPath('lib/pipeline/babel-loader-overrides.js');
```

Comment:
Will need to be changed to `lib/modules/pipeline/babel-loader-overrides.js`, and the `babel-loader-overrides.js` file will need to be moved from `lib/pipeline/` to `lib/modules/pipeline/`.

Reply:
Argh. Good catch! I missed that one